modules/gmc-agent/src/main/java/org/gridgain/agent/Agent.java

```diff
@@ -96,7 +96,7 @@
     private NodeConfigurationService nodeConfigurationSrvc;
 
     /** Execute service. */
-    private ExecutorService execSrvc;
+    private ThreadPoolExecutor execSrvc;
```

Comment:
Why we can not use `ExecutorService ` ?

Reply:
Because ExecutorService doesn't have getActiveCount method.